### DISCUSSION TITLE
Can I make orders on amazon using this & the MWS / SP-API?

### DISCUSSION THREAD

**Discussion by inspire22** [hakanensari/peddler#169] [Q&A]
I'm trying to place orders to buy and ship 1-3 copies of a book to ~300 different locations.  Is that something these APIs can do, or do you know of anything that might help?

The books are by me via amazon KDP, but I was expecting to have to use the public site & just lower the price.

Amazon's apis are pretty confusing, so looking for some guidance.

Thanks for a great project!

**Comment by hakanensari** [comment]
You probably would want to look into FBA, where you sell outside Amazon and have them fulfil orders for you.

Otherwise, no, you cannot programmatically place orders on Amazon through the seller APIs.
